docusaurus/src/components/Chip.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import styles from "./Chip.module.css";
+
+export const Chip = ({ className, ...rest }) => {
+  const chipClassName = `${styles.chip} ${className}`;
+  return <span className={chipClassName} {...rest} />;
+};
```

Comment:
```suggestion
import classNames from "classnames";

export const Chip = ({ className, children }) => (
  <span className={classNames(className, styles.chip)}>{children}</span>
);
```
nit: same classNames change, plus treat children the same way that we do in the other component for consistency